### PR TITLE
1567 Page directing new subscribers to confirm

### PR DESCRIPTION
**Pull request by dhochbaum-dcp**
### Summary
Create page that new subscribers are directed to, prompting them to check their emails for confirmation email.

- [x] Add new page to ZAP Search at route `/subscribed`
- [x] Build to specification for mobile and desktop in [Figma](https://www.figma.com/design/1HrOm1tkJ3lowQhrSRkPLs/ZAP-Listserv%3A-Wireframes-and-User-Stories?node-id=289-31&node-type=canvas&t=aqUGFpjHEPIyMRNp-0)
- [x] "Go to Home" links to ZAP Search home page (`/projects`).

#### Tasks/Bug Numbers
 - Completes #1567

### CODE REVIEW

File: client/app/router.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ Router.map(function() { // eslint-disable-line
   this.route('show-geography', { path: '/projects' });
   this.route('disclaimer');
   if (config.showAlerts) { this.route('statuses'); }
+  this.route('subscribed');
```
Comment (TylerMatteo) [inline client/app/router.js:32]: Should put this route behind the feature flag just like `statuses`.

File: client/app/templates/subscribed.hbs
Comment (TylerMatteo) [inline client/app/templates/subscribed.hbs:1]: Move styles to a sass file for the route.

**Review by TylerMatteo — CHANGES_REQUESTED** [review]

File: client/app/templates/subscribed.hbs
Comment (dhochbaum-dcp) [reply client/app/templates/subscribed.hbs:1]: Done

File: client/app/router.js (hunk 1 above)
Comment (dhochbaum-dcp) [reply client/app/router.js]: Done

**Review by TylerMatteo — APPROVED** [review]

**Pull request merged**